tests/unit/test_security.py

```
@@ -321,11 +353,20 @@ async def test_execute_with_invalid_working_dir(self):
         manager = PermissionManager()
         executor = SecureShellExecutor(manager)
 
-        # Use sleep command which should be allowed
-        success, stdout, stderr = await executor.execute("sleep 10", timeout=1)
+        # Use platform-appropriate sleep command
+        if platform.system() == "Windows":
+            # Windows timeout command (in seconds)
+            cmd = "timeout /t 10 /nobreak"
+        else:
+            # Unix sleep command
+            cmd = "sleep 10"
```

Ruff (0.11.9)

Use ternary operator `cmd = "timeout /t 10 /nobreak" if platform.system() == "Windows" else "sleep 10"` instead of `if`-`else`-block

(SIM108)